rerun_cpp/src/rerun.cpp

```diff
@@ -12,6 +12,67 @@ namespace rr {
         return rr_version_string();
     }
 
+    // ------------------------------------------------------------------------
+
+    RecordingStream RecordingStream::s_global;
```

Comment:
Not sure whether this is meant as a temporary solution, but ideally the C++ SDK would just bind to `global.rs` for global and thread-local recording management, like the python bindings, rather than doing its own thing right?

Reply:
We should definitely try to make the APis as similar as possible, and yes, this is only temporary.

I feel like we need to take yet another pass at these now that we are building for a single log function, i.e. one function for recordings _and_ for blueprints. Presumably that log function would send the data to different streams depending on the incoming type (is it recording data or blueprint data?), and so maybe we should hide all that "different streams for different things" stuff from the user somehow. I need to take a look at what Antoine and you have cooked up for `rerun2` in Python